knowledge_base/Contests.md

```diff
@@ -0,0 +1,39 @@
+# Contests
+
+Contests is a new feature available for communities on Common. This document outlines the implementation of Contests on our platform.
+
+## Basic Rules
+
+Each community may have only one contest running. Contests continue to run even if the prize pool is empty, but no payments will be made to winners in such cases.
+
+Admins cannot interfere with the content submission and voting processes. To be able to change constructor variables, admins must deploy a new contest contract with their new desired variables.
+
+On-chain data submission will be conducted and sponsored by Commonwealth Labs. The Common platform will not get any fee or share from the contests.
+
+Contests logic is managed by `Contest.sol`, and financial processes by the `FeeManager.sol` contract.
+
+## Contest Funding and Economics
+
+Contest prizes are primarily funded through the namespace’s Community Stake transaction commissions. This will be used by the `FeeManager.sol` contract, and CommunityStake fees generated from the community's staking activities will be directed to the contest contract. Additionally, direct transfers to the contest contract are also supported.
+
+Contest project will be able to utilize both ETH and ERC20 tokens. As of 240506, there is no additional funding plan for the contest project.
```

Comment:
Yep, has been YYMMDD since the get—I've seen this as an emerging standard in developer communities for datestamping because it avoids American vs International confusions (MMDDYY vs DDMMYY) and can be chronologically sorted using generic sort algorithms. See the "Indexicality & Timestamps" section of the Common Knowledge README (tho come to think of it, I should rename to "Indexicality & Datestamps" for clarity's sake).